examples/training_without_local_storage.ipynb

```diff
@@ -0,0 +1,917 @@
+    "\n",
+    "# %env AWS_ACCESS_KEY_ID=*****\n",
+    "# %env AWS_SECRET_ACCESS_KEY=***\n",
+    "# %env AWS_DEFAULT_REGION=us-west-2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In addition, if you didn't already, install Composer with streaming support:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install 'mosaicml[streaming,tensorboard]'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. The Local Training Workflow\n",
+    "\n",
+    "Let's first define our local training script for MNIST on the CIFAR-10 dataset. This training script downloads all data before training starts. We include the Tensorboard logger, which we will use to visualize results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import shutil\n",
+    "\n",
+    "import torch.utils.data\n",
+    "from torch.optim import SGD\n",
+    "from torchvision.datasets import MNIST\n",
+    "from torchvision.transforms import ToTensor\n",
+    "\n",
+    "from composer import Trainer\n",
+    "from composer.loggers import TensorboardLogger\n",
+    "from composer.utils.reproducibility import seed_all\n",
+    "from composer.models.classify_mnist import mnist_model\n",
+    "\n",
+    "seed_all(42)  # Set the seed before creating the model\n",
+    "\n",
+    "# Configure the trainer\n",
+    "model = mnist_model(num_classes=10)\n",
+    "optimizer = SGD(model.parameters(), lr=0.01)\n",
+    "train_dataset = MNIST('~/datasets', train=True, download=True, transform=ToTensor())\n",
+    "batch_size = 2048\n",
+    "train_dataloader = torch.utils.data.DataLoader(\n",
+    "    dataset=train_dataset,\n",
+    "    batch_size=batch_size,\n",
+    ")\n",
+    "eval_dataset = MNIST('~/datasets', train=False, download=True, transform=ToTensor())\n",
+    "eval_dataloader = torch.utils.data.DataLoader(\n",
+    "    dataset=eval_dataset,\n",
+    "    batch_size=batch_size,\n",
+    ")\n",
+    "\n",
+    "tensorboard_log_dir = '/tmp/tb_logs'\n",
+    "checkpoint_dir = '/tmp/checkpoints'\n",
+    "shutil.rmtree(tensorboard_log_dir, ignore_errors=True)\n",
+    "shutil.rmtree(checkpoint_dir, ignore_errors=True)\n",
+    "\n",
+    "trainer = Trainer(\n",
+    "    model=model,\n",
+    "    max_duration='2ep',\n",
+    "    # Make training fast: Terminate each epoch after 5 batches\n",
+    "    train_subset_num_batches=5,\n",
+    "    optimizers=optimizer,\n",
+    "    train_dataloader=train_dataloader,\n",
+    "    eval_dataloader=eval_dataloader,\n",
+    "    # Make evaluation fast: Evaluate only on five batches\n",
+    "    eval_subset_num_batches=5,\n",
+    "    # Flush log files every batch\n",
+    "    loggers=TensorboardLogger(log_dir=tensorboard_log_dir, flush_interval=1),\n",
```

Comment:
`flush_interval` of 1 is too small. it leads to dropping of points and points plotted out of order

Reply:
Hmm, I wasn't able to reproduce this issue:

![image](https://user-images.githubusercontent.com/87037432/183734815-6bc87207-ebc1-4b26-ac50-8785cdc4b2dd.png)